error-issue/index.ts

```diff
@@ -154,11 +154,8 @@ export async function errorMonitor(
   }
 }
 
-function createErrorReportUrl(report: ErrorReport): string {
-  const params = querystring.stringify({
-    ...report,
-    firstSeen: report.firstSeen.toString(),
-  });
+function createErrorReportUrl({errorId}: ErrorReport): string {
+  const params = querystring.stringify({errorId, linkIssue: 1});
```

Comment:
nit: linkIssueId

Reply:
This isn't an ID in this case, it's a boolean flag, using 1/0 because deserialization maintains truthiness/falsiness (based on convention used in error reporting).

Reply:
Oo ok.